doc/configuration/Hardware.md

```diff
@@ -0,0 +1,64 @@
+# Hardware
+
+*How to configure hardware and external endpoints that connect to HARMONI.*
+
+## Audio
+
+Check the `.asoundrc` file (located in `HARMONI/dockerfiles/config`) to ensure it refers to the appropriate sound device on your system. Determine which audio device you want to use by running `aplay -l` or `arecord -l` and update aforementioned `.asoundrc` file. *Note: make sure you can see hidden files*. If you are not using Docker, you will need to find or create the `.asoundrc` file in your home directory to configure sound in the same way.
+
+```bash
+cd dockerfiles/config
+nano .asoundrc
+# Update the hardware and soundcard id.
+```
+The contents are shown below:
+
+```
+pcm.!default {
+  type plug
+  slave {
+    pcm "hw:0,0"
+  }
+}
+ctl.!default {
+  type hw
+    card 0
+}
+```
+  
+The speaker device is controlled by the hw values, currently listed as 0,0 (the card `x` should be the same number of the first number in `hw:"x,y"`, and `y` refers to the device number).  
+To choose a speaker you may need to use the command `aplay -l `  to see what devices are available.  
+For example running `aplay -l`:
+```
+card 1: PCH [HDA Intel PCH], device 0: ALC283 Analog [ALC283 Analog]
+```
+In this case the `.asoundrc` file will be:
+```
+pcm.!default {
+  type plug
+  slave {
+    pcm "hw:1,0"
+  }
+}
+ctl.!default {
+  type hw
+    card 1
+}
+```
+
+### Testing Configuration
+
+After updating your asoundrc and restarting docker test the audio with the following command:
+```bash
+rostest harmoni_speaker speaker.test
+```
+If it works, you're good! Otherwise, re-do your [audio config](#audio), and try the above command until the audio works.
+
```

Comment:
Should we add also a reference to the microphone configuration?
Maybe I have missed it, but nowhere I could find any reference to the `configuration.yaml` file for microphone and camera which edits are fundamentals.

Reply:
I put in only what I knew of/was existing. If you can add more info either to the microphone package or to the hardware config, that would be great.